src/SimulationRunner.cc

```diff
@@ -1025,6 +1055,8 @@ void SimulationRunner::ProcessWorldControl()
     {
       this->requestedSeek = control.seek;
     }
+
+      this->SetRunToSimTime(control.runToSimTime);
```

Comment:
Nit: Extra indentation.

Reply:
Fixed in 9dd8abece910af6ef4e897447cf0b2aa8ca25663